examples/meta/src/evaluation/cross_validation_support_vector_machine.sg

```diff
@@ -1,5 +1,5 @@
-CSVFile f_feats("../../data/fm_train_real.dat")
-CSVFile f_labels("../../data/label_train_twoclass.dat")
+File f_feats = csv_file("../../data/fm_train_real.dat")
+File f_labels = csv_file("../../data/label_train_twoclass.dat")
 
 Features feats = features(f_feats)
 BinaryLabels labels(f_labels)
```

Comment:
@iglesias  There is a problem with StratifiedCrossValidationSplitting class. If you update to the new API, it leads to an error. Viktor told me he needs to discuss this with Heiko first since it is a design problem, so for now any label which is used with StratifiedCrossValidationSplitting has to use the old API (BinaryLabels)

Reply:
I changed this in #4313 to illustrate. You will have to rebase